Milestone 2

adriaanbd opened the pull request:
Additions:
* Components for App, Display, ButtonPanel and Button
* Functionless mockup of calculator with props

Link to deployed App on Heroku: https://reagir.herokuapp.com/

shloch commented on src/components/Display.js:
```diff
@@ -0,0 +1,20 @@
+Display.propTypes = {
+  result: PropTypes.string.isRequired,
+};
+
```
this solves the requirement of value type being `STRING`, but not deafaul value being ZERO.

search on `defaultProps` and implement 
https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png

adriaanbd replied:
Thank you. I'm not looking at the reviewer docs that must be why :laughing:!

adriaanbd replied:
I did not implement `defaultProps` as:

```

Display.defaultProps = {
  result: '0',
};
```

Because it isn't allowed with:

```
Display.propTypes = {
  result: PropTypes.string.isRequired,
};
```

See : 

```
propType "result" is required and should not have a defaultProps declaration.eslint(react/require-default-props)
```

shloch replied:
Yes you are right....it must be provided, but when it's not, the default value is automatically ZERO....like a normal calculator 🙂

adriaanbd commented:
Further changes:

* Changed `div` to `input` and added `disabled` option to it.

Reason:

![image](https://user-images.githubusercontent.com/22228397/70492035-16ba0c00-1ad1-11ea-99df-4bb1684aed40.png)

Info, here: https://reactjs.org/docs/dom-elements.html#value

adriaanbd commented:
Remember to check heroku deployed link on top if not cloning locally. 

Here's a screenshot:

![image](https://user-images.githubusercontent.com/22228397/70494685-3f92cf00-1ada-11ea-9eb9-97880218e77d.png)
